src/app/core/basic-datatypes/string/edit-text/display-long-text/display-long-text.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from "@angular/core";
+import { ViewDirective } from "../../../../entity/default-datatype/view.directive";
+import { DynamicComponent } from "app/core/config/dynamic-components/dynamic-component.decorator";
+
+@DynamicComponent("DisplayLongText")
+@Component({
+  selector: "app-display-long-text",
+  template: `<div [innerHTML]="formattedValue"></div>`,
+  standalone: true,
+})
+export class DisplayLongTextComponent extends ViewDirective<string, string> {
+  get formattedValue(): string {
```

Comment:
can you change this to a class variable instead of a get method and implement an [OnInit](https://v17.angular.io/api/core/OnInit) method where you calculate and set that variable only once?